engines/dfc_provider/app/serializers/dfc_provider/offer_serializer.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# Serializer used to render the DFC Offer from an OFN Product
+# into JSON-LD format based on DFC ontology
+module DfcProvider
+  class OfferSerializer
+    def initialize(variant)
+      @variant = variant
+    end
+
+    def serialized_data
+      {
+        "@id" => "offers/#{@variant.id}",
+        "@type" => "dfc:Offer",
+        "dfc:offeresTo" => {
+          "@type" => "@id",
+          "@id" => "/customerCategoryId1"
+        },
+        "dfc:price" => @variant.price,
+        "dfc:stockLimitation" => @variant.total_on_hand,
```

Comment:
the external api is on_hand, total_on_hand is internal and historical from spree.
I am sorry this is not clearly documented anywhere :-(